code/game/objects/items/tcg/tcg.dm

```diff
@@ -130,10 +140,16 @@ GLOBAL_LIST_EMPTY(cached_cards)
 
 /obj/item/cardpack/equipped(mob/user, slot, initial)
 	. = ..()
-	transform = matrix()
+	zoom_in()
 
 /obj/item/cardpack/dropped(mob/user, silent)
 	. = ..()
+	zoom_out()
+
+/obj/item/cardpack/proc/zoom_in()
+	transform = matrix()
+
+/obj/item/cardpack/proc/zoom_out()
 	transform = matrix(0.4,0,0,0,0.4,0)
 
```

Comment:
```suggestion
/obj/item/cardpack/proc/zoom_out()
	transform = matrix(0.4,0,0,0,0.4,0)
	UnregisterSignal(src, list(COMISG_STORAGE_ENTERED, CONSIG_STORAGE_EXITED))

```

Reply:
why unregister here?

Reply:
It's not unregistering anywhere.

Reply:
why does it need to unregister? do I need to do it on Destroy() or something?

Reply:
Guess I misread it.